README.md

```diff
@@ -15,12 +15,291 @@ Download tarball in the release page and extract it to your `$HOME/.julia` direc
 
 ### build from source
```

Comment:
it'd be nice if this can be in a separate PR, I probably will put up the website soon, so we can decide whether to have this in a different PR